import.go

```diff
@@ -48,6 +49,7 @@ func importBuilderDataFromImage(ctx context.Context, store storage.Store, system
 	}
 
 	var instanceDigest *digest.Digest
```

Comment:
I think this variable becomes unused in this scope.  It can be implicitly declared where it's still used, or removed entirely.

Reply:
`instanceDigest` is used for reporting errors from `image.FromUnparsedImage`, it is used in both the single-platform and multi-platform case (but only in case of an error).

(The idea is that `instanceDigest` and `unparsedInstance` should match, i.e. `unparsedInstance == image.UnparsedInstance(srcSrc, instanceDigest)`.)

Reply:
Ah, I missed it being used in adding context to the error message.